CONTRIBUTING.adoc

```diff
@@ -17,8 +17,8 @@ Here is the expected commit format, please follow https://www.conventionalcommit
 
 == Pull request
 
-Each improvment, fix must be submit with a pull request. When a pull request is submit, a action is running to check if your pull request follow commit message format.
+Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is running to check if your pull request follow commit message format.
```

Comment:
```suggestion
Each improvement, fix must be submit with a pull request. When a pull request is submit, an action is run to check if the pull request follows the commit message format.
```

Reply:
For each improvement, the fix must be submitted with a pull request. When a pull request is submitted, an action is run to check if the pull request follows the commit message format.